GCMessengerSDKSample/build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.0'
+        classpath 'com.android.tools.build:gradle:8.0.2'
```

Comment:
what will this upgrade require from users that are currently working with this app on lower gradle version?

Reply:
yes, it will require the users to use the gradle version that is now specified.